public/app/core/components/Breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -61,9 +65,21 @@ const getStyles = (theme: GrafanaTheme2) => {
       color: theme.colors.text.primary,
       display: 'flex',
       fontWeight: theme.typography.fontWeightMedium,
+
+      // logic for small screens
+      // hide any breadcrumbs that aren't the second to last child (the parent)
+      [theme.breakpoints.down('md')]: {
+        display: 'none',
+        '&:nth-last-child(2)': {
+          display: 'flex',
+          flexDirection: 'row-reverse',
+
+          [`.${separator}`]: {
```

Comment:
not sure completely about this - emotion docs were a little vague about how best to accomplish targetting another class? 🤔

Reply:
Hmm it seems to work... so I guess we keep it? 🤷 I had a quick google and doesn't seem like there is an easier way for it